docs/rules/no-deprecated.md

```diff
@@ -22,6 +22,8 @@ React.createClass({ /* Class object */ });
 const propTypes = {
   foo: PropTypes.bar,
 };
+
+import {React PropTypes} from 'react';
```

Comment:
That would import a named import called "React", which doesn't exist.

The syntax I supplied is the correct one.